content/articles/semantic-html-for-beginners/index.md

```diff
@@ -0,0 +1,403 @@
+                     ullam odio inventore vitae ab in, itaque possimus rerum sapiente doloribus
+                      vel reiciendis soluta temporibus dicta. Maiores dignissimos nihil quod ex
+                       eveniet, voluptatem necessitatibus dolorum, ipsum excepturi inventore omnis
+                       suscipit dolorem pariatur? Minima voluptatibus rerum error laboriosam.
+                       Repellendus, quae eligendi. Sed neque hic cupiditate quam quia eius animi
+                        delectus accusantium iusto asperiores alias quibusdam, qui recusandae
+                        necessitatibus maxime cum? Voluptas, consequatur. Nisi est corporis
+                         quasi vero, nam esse magnam. Dolores iusto debitis nostrum magnam.
+                </p>
+            </article>
+
+        </section>
+
+        <aside>
+            <article>
+                <h3>Join Our newsletter</h3>
+                <br>
+            <input type="text" placeholder="Your email">
+
+            </article>
+            <br>
+            <hr>
+            <br>
+
+            <article>
+                <h3>Contact Us</h3>
+                <p>
+                </p>
+                Lorem ipsum dolor, sit amet consectetur
+                adipisicing elit. Voluptatem minima dolor
+                rem, perferendis tempore laborum mollitia
+                voluptatum maxime quos ducimus, iure adipisci
+                labore magnam atque quis tenetur quae, omnis voluptas?
+            </p>
+
+               </article>
+        </aside>
+
+    </main>
+
+    <footer>
+        &copy; <b>2021 Saxophone Shop</b>
+    </footer>
+
+</body>
+</html>
+```
+
+What you should know:
+
+1. Semantic HTML is just about given meaningful names to your HTML structure.
+1. The semantic tag names cannot be edited.
+
+For instance:
+
+`<header>` will always be `<header>`
+
+What's happen here?
+
+1. we used up to five semantic tags to drive the importance of using semantics.
+1. the `header` tag is for the name of the project or website.
+1. the `nav` tag will guide the new developer to know that the nav stands for navigation (where all the useful links reside).
+1. we have the middle area which is called the `main` tag in HTML5.
+1. we have the `footer` tag showcasing the closing section of the web page.
+1. in the `main` area we can have a lot of things inside. Semantic HTML comes baked with its special tag some of them include `section`, `article`, and `aside` tags.
+1. the `section` area showcase different sections of our web page. Multiple sections can be created. the `section` can be in the `header`, `footer`.
+1. the `aside` area showcase the area of our web page not too relevant. It can also serve as the sidebar.
+
+Finally, these are the basic semantic HTML5 tags, which can make our HTML more readable and easier to understand. However, it does not affect the view of the web page.
+
+### Conclusion
+

```

Comment:
The conclusion is a recap of the article.
So give a short summary of what your article has covered (Keep it two to three sentences)